### PR TITLE
ui updates

### PR DESCRIPTION
**Pull request by mikefortney**
moved footer from individual pages to layout component (except for homepage which is a one-off), added links to expanded leaderboard row, added background to header

### CODE REVIEW

File: components/Leaderboard/Leaderboard.js [hunk 1]
```diff
@@ -6,6 +6,8 @@ import { useRouter } from "next/router";
 // A super simple expandable component.{JSON.stringify(data, null, 2)}
 const ExpandedComponent = ({ data }) => {
   const router = useRouter();
+  let links = data.helpfulLinks.split(",");
```
Comment (tomknig) [inline components/Leaderboard/Leaderboard.js:9]: I think this threw an exception for me when testing:
image

Error occurred when I was expanding one of the entries on the leaderboard.

Comment (mikefortney) [reply components/Leaderboard/Leaderboard.js:9]: ok thx i'll look into this

File: components/Leaderboard/Leaderboard.js [hunk 2]
```diff
@@ -6,8 +6,10 @@ import { useRouter } from "next/router";
 // A super simple expandable component.{JSON.stringify(data, null, 2)}
 const ExpandedComponent = ({ data }) => {
   const router = useRouter();
-  let links = data.helpfulLinks.split(",");
-
+  let links = [];
+  if (data.helpfulLinks) {
+    links = data.helpfulLinks.split(",");
+  }
```
Comment (tomknig) [inline components/Leaderboard/Leaderboard.js:12]: This is good. You could also utilize optional chaining and nullish coalescing here. I am not saying that it's better, just slightly more idiomatic for me. But as said, your implementation is fine and approved :slightly_smiling_face: 
`const links = data.helpfulLinks?.split(",") ?? [];`

**Review by tomknig — APPROVED** [review]

**Comment by kurdybacha** [comment]
Nice!

**Pull request merged**